gallery/src/components/NxIconDropdown/NxIconDropdownPage.tsx

```diff
@@ -22,7 +22,7 @@ const NxIconDropdownPage = () =>
       <NxP>
         A variation of the dropdown menu component which uses a single icon as its toggle.
         <NxCode>NxIconDropdown</NxCode> shares all of the same props as{' '}
-        <NxTextLink href="#/pages/NxDropdown" className="nx-dropdown-button">
+        <NxTextLink href="#/pages/Dropdown" className="nx-dropdown-button">
```

Comment:
The `.nx-dropdown-button` class shouldn't be here, let's go ahead and remove that

Reply:
> The `.nx-dropdown-button` class shouldn't be here, let's go ahead and remove that

Fixed here [1ebf468](https://github.com/sonatype/sonatype-react-shared-components/pull/721/commits/1ebf468a967985ef012b09748741ff63da0576c5)